packages/oui-numeric/src/numeric.controller.js

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash'
```

Comment:
@neolitec You have integrated to tool where we can import lodash by part like `import { clamp } from 'lodash'` ?

Reply:
`babel-plugin-lodash` in combination with `lodash-webpack-plugin` can help cleaning the dist of unused lodash parts. It's already in the project.